planner/core/util.go

```diff
@@ -295,6 +295,32 @@ func extractStringFromStringSet(set set.StringSet) string {
 	return strings.Join(l, ",")
 }
 
+// extractStringFromIntSet helps extract string info from set.Int64Set.
+func extractStringFromIntSet(set set.Int64Set) string {
```

Comment:
```suggestion
func extractStringFromInt64Set(set set.Int64Set) string {
```

Reply:
This function is removed when I remove extraction of `table_id` and `index_id`  columns this morning.  
At the same time, below columns are removed too:
*  Other columns filtered on TiDB: `db_name`, `table_name`, `index_name` .
* Range condition of `hot_degree`, `flow_bytes`,  `key_rate`, `query_rate` .